src/pks/transport/transport_ats_pk.cc

```diff
@@ -199,6 +201,9 @@ Transport_ATS::SetupTransport_()
   num_advect_ = plist_->get<int>("number of aqueous components advected", num_aqueous_);
   num_gaseous_ = plist_->get<int>("number of gaseous components", 0);
 
+  tcc_max_.resize(num_aqueous_, 1.e10);
+  tcc_max_ = plist_->get<Teuchos::Array<double>>("component max concentrations", tcc_max_);
```

Comment:
This should get added to documentation too, but the transport documentation needs a lot of help, so don't worry too much about this one.